ice.c

```diff
@@ -2516,7 +2516,7 @@ static void janus_ice_cb_nice_recv(NiceAgent *agent, guint stream_id, guint comp
 							retransmits_cnt++;
 							/* Enqueue it */
 							janus_ice_queued_packet *pkt = g_malloc(sizeof(janus_ice_queued_packet));
-							pkt->data = g_malloc(p->length);
+							pkt->data = g_malloc(p->length+SRTP_MAX_TAG_LEN);
```

Comment:
This actually should only happen if we're using RFC4588, as in that case we encrypt again: if not, we're sending an already encrypted packet, and so `srtp_protect` is not done.

Reply:
Thinking about it, it's really not that important... I can add that later on, and the additional checks you added are important enough to merge them ASAP. Thanks!